packages/react-scripts/config/webpack/persistentCache/createEnvironmentHash.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const { createHash } = require('crypto');
+
+module.exports = env => {
+  const hash = createHash('md4');
```

Comment:
Is this a typo or do we really want `md4` here?

Reply:
The file hash webpack 5 uses is md4 afaik - we could use md5 it's just a content hash of the environment settings for invalidation of the new filesystem cache in webpack 5

Reply:
Updated to use md5 - but it's a minor detail again related to generate a content hash of the environment config for webpack 5 cache invalidation